docs/source/tutorials/playing-command.rst

```diff
@@ -76,22 +84,35 @@ Now the command will be able to react a bit more to your user::
     <YourNick> .hello noon
     <Sopel> Hello YourNick, are you having lunch?
 
+How does that work? Well, the short version is that Sopel uses regex
+(`Regular Expression`__) to match a message to a plugin callable, and the
```

Comment:
```suggestion
(`REGular EXpressions`__) to match a message to a plugin callable, and the
```

If you can bold-face (**Reg**ular **Ex**pressions) or otherwise emphasize the relevant parts of each word within the link markup, feel free to do that instead; I just don't trust my reST skills enough to suggest that blind. 😁

Reply:
Good call! I'll see what I can do with the **boldening**.